crux-bench/cloudformation.yaml

```diff
@@ -199,6 +199,9 @@ Resources:
             - Name: "SLACK_URL"
               ValueFrom:
                 "arn:aws:secretsmanager:eu-west-2:955308952094:secret:bench/slack-url-uumMHQ"
+          Environment:
```

Comment:
Given that we're only adding this here (and not within `crux/run-bench`), am I correct in understanding that by default we're only ever collecting these metrics for the master `crux-bench` runs, and not for the `crux-bench-dev` runs?

Reply:
Yep :)